packages/client/src/components/App/App.tsx

```diff
@@ -15,11 +30,29 @@ function App() {
           <Route index element={<Welcome />} />
           <Route path={login} element={<SignIn />} />
           <Route path={registration} element={<SignUp />} />
+          <Route path={page500} element={<div>500</div>} />
           <Route path="*" element={<div>404</div>} />
-
+ 
           {/* Приватные */}
           <Route path={game} element={<PrivateRoute />}>
-            <Route index element={<div>main game page</div>} />
+            <Route index element={<div>Страница начала игры</div>} />
+            <Route path={play} element={<div>Страница игры(canvas)</div>} />
+            <Route path={end} element={<div>Страница конца игры</div>} />
+          </Route>
+          <Route path={forum} element={<PrivateRoute />}>
+            <Route index element={<div>Страница со списком всех тем</div>} />
+            <Route path={newThread} element={<div>Страница создания новой темы</div>} />
+            <Route path={thread} element={<div>Страница с обсуждениями темы1</div>}>
+              <Route path={editThread} element={<div>Страница изменения темы</div>} />
+            </Route>
+          </Route>
+          <Route path={leaderboard} element={<PrivateRoute />}>
+            <Route index element={<div>Лидербоард</div>} />
+          </Route>
```

Comment:
Я пропустил ранее, но мне кажется, что приватные роуты можно организовать иначе.

Все таки <Outlet/>, который вы используете для приватных роутов, предназначен скорее для рендера контента вложенных роутов.

Как смотрите на то, чтобы приватные роуты обернуть хоком примерно вот так: 

```suggestion
          <Route path={leaderboard} element={
              <RequireAuth>
                  <div>Лидербоард</div>
               </RequireAuth>
           }/>
```

Reply:
НА обсуждение команде короче )

Reply:
у нас с командой сегодня будет митинг, обязательно обсудим!

Reply:
>Как смотрите на то, чтобы приватные роуты обернуть хоком примерно вот так:

В чем преимущество? Какая разница?